tool/env-set.sh

```diff
@@ -57,11 +57,11 @@ elif [[ -z "$DART_SITE_ENV_DEFS" ]]; then
   fi
   source tool/shared/get-ruby.sh "$_DART_SITE_ENV_SET_INSTALL_OPT"
 
-  if ! __type_t travis_fold > /dev/null; then
-    # In case this is being run locally. Turn travis_fold into a noop.
-    travis_fold() { true; }
-  fi
-  export -f travis_fold > /dev/null
+  # if ! __type_t travis_fold > /dev/null; then
+  #   # In case this is being run locally. Turn travis_fold into a noop.
+  #   function travis_fold() { true; }
+  # fi
+  # export -f travis_fold > /dev/null
```

Comment:
Would it be better if I remove here ?

Reply:
Yeah, just remove rather than comment.

Reply:
We have the git history if needed.

Reply:
Thanks for the review.
Removed comment.